great_expectations/expectations/core/expect_column_values_to_match_like_pattern_list.py

```diff
@@ -126,19 +127,58 @@ def validate_configuration(
             raise InvalidExpectationConfigurationError(str(e))
 
     @classmethod
-    @renderer(renderer_type=LegacyRendererType.PRESCRIPTIVE)
-    @render_evaluation_parameter_string
+    @renderer(renderer_type="renderer.prescriptive")
     def _prescriptive_renderer(
         cls,
         configuration: Optional[ExpectationConfiguration] = None,
         result: Optional[ExpectationValidationResult] = None,
         runtime_configuration: Optional[dict] = None,
         **kwargs,
-    ) -> None:
+    ) -> List[RenderedStringTemplateContent]:
         runtime_configuration = runtime_configuration or {}
         _ = False if runtime_configuration.get("include_column_name") is False else True
-        _ = runtime_configuration.get("styling")
-        params = substitute_none_for_missing(  # noqa: F841 # unused
+        styling = runtime_configuration.get("styling")
+
+        params = substitute_none_for_missing(
             configuration.kwargs,
-            ["column", "mostly", "row_condition", "condition_parser"],
+            ["column", "like_pattern_list", "mostly"],
         )
+        if params["mostly"] is not None:
+            params["mostly_pct"] = num_to_str(
+                params["mostly"] * 100, precision=15, no_scientific=True
+            )
+
+        if (
+            not params.get("like_pattern_list")
+            or len(params.get("like_pattern_list")) == 0
+        ):
+            values_string = "[ ]"
+        else:
+            for i, v in enumerate(params["like_pattern_list"]):
+                params[f"v__{i!s}"] = v
+            values_string = " ".join(
+                [f"$v__{i!s}" for i, v in enumerate(params["like_pattern_list"])]
+            )
+
+        template_str = "Values must match the following like patterns: " + values_string
+
+        if params["mostly"] is not None and params["mostly"] < 1.0:  # noqa: PLR2004
+            params["mostly_pct"] = num_to_str(
+                params["mostly"] * 100, precision=15, no_scientific=True
```

Comment:
This was sortof already done, but I think it can be cleaned up as part of the PR. Thank you for the suggestion here
https://github.com/great-expectations/great_expectations/blob/d9b8180fece6e285f70e0df20c295612b23a31d1/great_expectations/render/util.py#L18